tensorflow_probability/python/distributions/laplace_test.py

```diff
@@ -340,5 +342,31 @@ def testLaplaceNonPositiveInitializationParamsRaises(self):
           loc=loc_v, scale=scale_v, validate_args=True)
       self.evaluate(laplace.mean())
 
+  @tfe.run_test_in_graph_and_eager_modes()
+  def testLaplaceLaplaceKL(self):
+    batch_size = 6
+    event_size = 3
+
+    a_loc = np.array([[0.5] * event_size] * batch_size, dtype=np.float32)
+    a_scale = np.array([[0.1] * event_size] * batch_size, dtype=np.float32)
+    b_loc = np.array([[0.4] * event_size] * batch_size, dtype=np.float32)
+    b_scale = np.array([[0.2] * event_size] * batch_size, dtype=np.float32)
+
+    a = laplace_lib.Laplace(loc=a_loc, scale=a_scale)
+    b = laplace_lib.Laplace(loc=b_loc, scale=b_scale)
+
+    distance = tf.abs(a_loc - b_loc)
+    ratio = a_scale / b_scale
+    true_kl = -tf.log(ratio) - 1 + distance / b_scale + \
```

Comment:
Nit: Prefer parentheses to newline escaping:

```suggestion
    true_kl = (-tf.log(ratio) - 1 + distance / b_scale +
                     ratio * tf.exp(-distance / a_scale))
```